compute/resource_job.go

```diff
@@ -222,6 +222,8 @@ var jobSchema = common.StructToSchema(JobSettings{},
 		}
 		s["email_notifications"].DiffSuppressFunc = common.MakeEmptyBlockSuppressFunc("email_notifications.#")
 		s["max_concurrent_runs"].ValidateDiagFunc = validation.ToDiagFunc(validation.IntAtLeast(1))
+		s["max_concurrent_runs"].Optional = true
+		s["max_concurrent_runs"].Default = 1
```

Comment:
Rebased and set `Optional` and `Default` here. Is that what you're thinking?

Reply:
No need to set `Optional` as it's coming from `omitempty` - https://github.com/databrickslabs/terraform-provider-databricks/blob/master/compute/model.go#L553, which is parsed through reflection. Small trick to keep the size of the codebase manageable.